ARCs/arc-0034.md

```diff
@@ -0,0 +1,96 @@
+---
+arc: 34
+title: Governance proposal process
+description: Criteria for the creation of proposals.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Living
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+The goal of this ARC is to increase transparency and efficiency in the decision-making process by clearly defining the steps involved and ensuring that all proposals are given proper consideration. 
```

Comment:
The Goal of this ARC is to clearly define the steps involved in submitting proposals for the xGov Program, to increase transparency and efficiency, ensuring all proposals a given proper consideration.